docs/luigi-compound-container-api.md

```diff
@@ -16,7 +16,9 @@ meta -->
 
 # Luigi Compound Container API
 
-This document outlines the parameters provided by the Luigi Compound Container. Luigi Compound Container provides the possibility to insert multiple webcomponent-based microfrontends in one container
+This document outlines the parameters provided by the Luigi Compound Container. Luigi Compound Container provides the possibility to insert multiple webcomponent-based microfrontends in one containe.</br>
+In addition you can use standard `addEventListener` function to react on events emmitted by the luigi compound container. The list of events and their meaning can be found [here](https://github.com/SAP/luigi/blob/main/container/src/constants/communication.ts).
```

Comment:
Perhaps capitalized version would be more suitable? This pattern is used in the sentence above:
'Luigi Compound Container'

Reply:
thx